src/socket-message-handling.ts

```diff
@@ -71,6 +90,8 @@ const setupWSConnection = (conn) => {
     });
     subscribedTopics.clear();
     closed = true;
+    conn.close();
```

Comment:
This seems pretty circular - the y-websocket server file had a call to closeConn(doc, conn) here which I assume is mainly for disconnecting from the doc?